examples/hybrid/sphere/hadley_circulation.jl

```diff
@@ -235,10 +235,15 @@ parameters = (; coords, face_coords, τ, u₀, w₀, K, ρ₀, ystar, y_td)
 rhs!(ystar, y0, parameters, 0.0, dt, 1)
 
 # run!
-prob = ODEProblem(IncrementingODEFunction(rhs!), copy(y0), (0.0, T), parameters)
+prob = ODEProblem(
+    ClimaODEFunction(; T_exp != rhs!),
```

Comment:
```suggestion
    ClimaODEFunction(; T_exp! = rhs!),
```
I think the white space was misplaced here!